src/app/locationForm/description/LocationDescription.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
It looks like this is actually supposed to be the organization description, not location description.
In the most recent designs, at least, the text is "How would you describe this organization?". (And also, in our initial data we only have descriptions of organization, no locations.)

Reply:
Fixed in 8383dbc